.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*]
+indent_size = 2
+indent_style = space
```

Comment:
Not really part of the PR

Reply:
Do you really oppose adding this? Without editorconfig the default formatting was all wring in my IDE